# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/quota/CommonMetadata.h
# Commit: f466599094b7
# Full Hash: f466599094b7e16948794a49b0571e03eb2b4df9
# Author: Serban Stanca <sstanca@mozilla.com>
# Date: 2025-06-15 13:49:48
# Description:
#   Revert "Bug 1962133 - QM: Add accessed flag to FullOriginMetadata and persist it in the origin metadata file; r=dom-storage-reviewers,hsingh" as requested by dmeehan for causing top crash in Bug 1972221.
#   
#   This reverts commit 433768e64dd71c996694e08005854756b7c45fb3.
# ==============================================================================

diff -r 2408bee8bcc4 -r f466599094b7 dom/quota/CommonMetadata.h
--- a/dom/quota/CommonMetadata.h	Sat Jun 14 05:48:43 2025 +0000
+++ b/dom/quota/CommonMetadata.h	Sun Jun 15 16:46:59 2025 +0300
@@ -80,15 +80,12 @@
 
 struct OriginStateMetadata {
   int64_t mLastAccessTime;
-  bool mAccessed;
   bool mPersisted;
 
   OriginStateMetadata() = default;
 
-  OriginStateMetadata(int64_t aLastAccessTime, bool aAccessed, bool aPersisted)
-      : mLastAccessTime(aLastAccessTime),
-        mAccessed(aAccessed),
-        mPersisted(aPersisted) {}
+  OriginStateMetadata(int64_t aLastAccessTime, bool aPersisted)
+      : mLastAccessTime(aLastAccessTime), mPersisted(aPersisted) {}
 };
 
 struct FullOriginMetadata : OriginMetadata, OriginStateMetadata {